GVFS/GVFS.Notifications/VFSForGit.Mac/StatusMenuItem/UI/AboutWindowController.xib

```diff
@@ -43,7 +43,7 @@
                                         <color key="backgroundColor" name="controlColor" catalog="System" colorSpace="catalog"/>
```

Comment:
For my own understanding, how did you create this file?

Reply:
This file is created by Xcode. When creating a `Controller` class for `UI`, Xcode provides an option to specify a `xib` file as well. 

<img width="761" alt="Screen Shot 2019-05-24 at 3 10 42 PM" src="https://user-images.githubusercontent.com/378580/58351304-69362280-7e36-11e9-996a-c24bcb0a7667.png">